packages/react-components/review/stable/react-components.api.md

```diff
@@ -236,6 +236,9 @@ export interface ComponentStrings {
     devicesButton: DevicesButtonStrings;
     endCallButton: EndCallButtonStrings;
     errorBar: ErrorBarStrings;
+    fileCard: FileCardStrings;
```

Comment:
This should not be in stable API

Reply:
these strings are added from json file, cannot conditionally compile these.